chainer/functions/pooling/unpooling_2d.py

```diff
@@ -43,7 +43,7 @@ def forward(self, x):
         xp = cuda.get_array_module(*x)
         col = xp.tile(x[0][:, :, None, None],
                       (1, 1, self.kh, self.kw, 1, 1))
-        if isinstance(x[0], cuda.ndarray):
```

Comment:
`cuda.cupy` is unavailable on environments without a GPU. How about `xp is not numpy`?